devtools/cli/src/test/java/io/quarkus/cli/CliDriver.java

```diff
@@ -311,7 +311,7 @@ public static Result invokeValidateDryRunBuild(Path projectRoot) throws Exceptio
 
     public static Result invokeValidateBuild(Path projectRoot) throws Exception {
         Result result = execute(projectRoot, "build", "-e", "-B", "--clean",
-                "-Dproperty=value1", "-Dproperty2=value2");
```

Comment:
is that change necessary here ?

Reply:
Yes. I am verifying the properties go through properly.

Reply:
--verbose? not necessarily.. other than we didn't know what was going on ..